core/services/ocr2/plugins/s4/factory.go

```diff
@@ -33,7 +33,7 @@ func (f S4ReportingPluginFactory) NewReportingPlugin(rpConfig types.ReportingPlu
 		UniqueReports: false,
 		Limits:        *limits,
 	}
-	plugin, err := NewReportingPlugin(f.Logger, config, f.ORM)
+	plugin, err := NewReportingPluginInstance(f.Logger, config, f.ORM)
```

Comment:
I'd avoid changing that for consistency with other factories. The "Instance" suffix doesn't add any information anyway.